### PR TITLE
Add BrokerRequest layer

### PR DESCRIPTION
**Pull request by djino97**

### CODE REVIEW

File: src/NewsService.Models.BrokerRequest/NewsService.Models.BrokerRequest.csproj [hunk 1]
```diff
@@ -0,0 +1,9 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.1</TargetFramework>
+    <AssemblyName>LT.DigitalOffice.NewsService.Models.BrokerRequest</AssemblyName>
+    <RootNamespace>LT.DigitalOffice.NewsService.Models.BrokerRequest</RootNamespace>
```
Comment (KostinP) [inline src/NewsService.Models.BrokerRequest/NewsService.Models.BrokerRequest.csproj:6]: Not "LT.DigitalOffice.Models.BrokerRequest"?

File: src/NewsService.Models.BrokerRequest/Class1.cs [hunk 2]
```diff
@@ -0,0 +1,8 @@
+﻿using System;
+
+namespace LT.DigitalOffice.Broker.Request
+{
+    public class Class1
+    {
+    }
+}
```
Comment (KostinP) [inline src/NewsService.Models.BrokerRequest/Class1.cs:8]: Extra class

File: LT.DigitalOffice.NewsService.sln [hunk 3]
```diff
@@ -1,38 +1,41 @@
...
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "NewsService.Mappers.UnitTests", "test\NewsService.Mappers.UnitTests\NewsService.Mappers.UnitTests.csproj", "{9A11D177-89C3-43A5-8A55-03C506A69734}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "NewsService.Broker.UnitTests", "test\NewsService.Broker.UnitTests\NewsService.Broker.UnitTests.csproj", "{952FECB2-0BC2-441C-8D1D-F2B7C63CD070}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "NewsService.Broker.UnitTests", "test\NewsService.Broker.UnitTests\NewsService.Broker.UnitTests.csproj", "{952FECB2-0BC2-441C-8D1D-F2B7C63CD070}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "NewsService.Data.UnitTests", "test\NewsService.Data.UnitTests\NewsService.Data.UnitTests.csproj", "{93B126D0-DF48-4B9B-BE5C-8D973B0CD26E}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "NewsService.Data.UnitTests", "test\NewsService.Data.UnitTests\NewsService.Data.UnitTests.csproj", "{93B126D0-DF48-4B9B-BE5C-8D973B0CD26E}"
```
Comment (KostinP) [inline LT.DigitalOffice.NewsService.sln:34]: Why new GUID?

File: src/NewsService.Models.BrokerRequest/Class1.cs (hunk 2 above)
Comment (djino97) [reply src/NewsService.Models.BrokerRequest/Class1.cs:8]: This is for example

File: LT.DigitalOffice.NewsService.sln (hunk 3 above)
Comment (djino97) [reply LT.DigitalOffice.NewsService.sln:34]: This is automatically updated

**Review by lalagvana — APPROVED** [review]

**Review by DarkFelas — APPROVED** [review]

**Pull request merged**